src/main/java/org/fundacionjala/coding/melvi/MovieRefactor/PriceCalculationRegularClient.java

```diff
@@ -0,0 +1,40 @@
+package org.fundacionjala.coding.melvi.MovieRefactor;
+
+/**
+ *
+ */
+public class PriceCalculationRegularClient implements IPriceCalculation {
+
+    public static final int THREE = 3;
+    public static final double ONEPOINTFIVE = 1.5;
+
+    /**
+     * @param rentedMovie Movie rented
+     * @param daysRented  days it was rented
+     * @return price
+     */
+    public double getPrice(final Movie rentedMovie, int daysRented) {
+        double amount = 0;
+        switch (rentedMovie.getPriceCode()) {
+            case REGULAR:
+                amount = 2;
+                if (daysRented > 2) {
+                    amount += (daysRented - 2) * ONEPOINTFIVE;
+                }
+                break;
+            case NEW_RELEASE:
+                amount = daysRented * THREE;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "double". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)